controllers/v1/solutions.js

```diff
@@ -93,7 +93,6 @@ module.exports = class Solutions extends Abstract {
 		return new Promise(async (resolve, reject) => {
```

Comment:
@Prajwal17Tunerlabs Please update api doc where ever we made request body changes pls

Reply:
api docs updated.